File: VB/WebApplication1/IService1.vb
Imports System
Imports System.Collections
Imports System.Collections.Generic
Imports System.Linq
Imports System.Runtime.Serialization
Imports System.ServiceModel
Imports System.Text

Namespace WebApplication1
    ' NOTE: You can use the "Rename" command on the "Refactor" menu to change the interface name "IService1" in both code and config file together.
    <ServiceContract> _
    Public Interface IService1
        <OperationContract, ServiceKnownType(GetType(Item))> _
        Function GetData(ByVal skipCount As Integer, ByVal takeCount As Integer) As IEnumerable
        <OperationContract> _
        Function Count() As Integer
    End Interface
    Public Interface IItem
        Property Id() As Integer
        Property Name() As String
    End Interface
End Namespace

File: CS/WebApplication1/IService1.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.Text;

namespace WebApplication1 {
    // NOTE: You can use the "Rename" command on the "Refactor" menu to change the interface name "IService1" in both code and config file together.
    [ServiceContract]
    public interface IService1 {
        [OperationContract]
        [ServiceKnownType(typeof(Item))]
        IEnumerable GetData(int skipCount, int takeCount);
        [OperationContract]
        int Count();
    }
    public interface IItem {
        int Id { get; set; }
        string Name { get; set; }
    }
}